### PR TITLE
update: proguard configuration

### PR DESCRIPTION
**Pull request by tasomaniac**
- Crashlytics is added.
- Proper and updated configs for okhttp, okio and butterknife
- ACRA removed

### CODE REVIEW

**Comment by tasomaniac** [comment]
I will also merge this if no one says not to :)

File: app/proguard-rules.txt [hunk 1]
```diff
@@ -68,49 +79,17 @@
...
-# keep this otherwise it is removed by ProGuard
--keep public class org.acra.ErrorReporter
-{
-    public void handleSilentException(java.lang.Throwable);
+-keepclasseswithmembernames class * {
+    @butterknife.* <methods>;
 }
 
-# GMS
+#GMS
```
Comment (Splaktar) [inline app/proguard-rules.txt]: Not sure why this was needed. There seems to be inconsistent spacing after `#` throughout the file. I prefer the space as do the snippets here (https://github.com/krschultz/android-proguard-snippets/tree/master/libraries).

Comment (tasomaniac) [inline app/proguard-rules.txt]: Thank you for the review. I will update it. Not sure why this happened. :)

Comment (tasomaniac) [inline app/proguard-rules.txt]: Do you think that we should use separate files for every library project we have? Where should we keep those files?

**Comment by tasomaniac** [comment]
Just updated the PR. Can you take a look. ty.

**Pull request merged**